### DISCUSSION TITLE
German Translation

### DISCUSSION THREAD

**Discussion by zhutoutoutousan** [vuejs-translations/guidelines#22] [Ideas]
Well, I'll be damned, I'm a native 'noobesten' developer & Chinese English, German, ... speaker here, and I translated tons of German-Chinese, English-Chinese Subtitles, and documents.
I used MemoQ, SDL Trados stuffs, this software can batch replace words or sentences according to termbase, anybody got an idea on using that?

- [Integration with Git-based repositories Revamp Your Workflow with the memoQ Ecosystem](https://www.memoq.com/integrations/content-management/git-based-repositories)

**Comment by Seuthes** [comment]
I could also help out with a german translation.

**Reply by janisal98** [reply]
I would like to help you with the translation. How is it going forward? 
Sollen wir uns mal zusammen hinsetzen?

**Comment by roma-marshall** [comment]
### If anyone wants to help with translation, please switch to an active discussion
### Wenn jemand bei der Übersetzung helfen möchte, bitte zu einer aktiven Diskussion wechseln

> ### [German Translation Discussion](https://github.com/vuejs-translations/guidelines/discussions/49)

@zhutoutoutousan please mark as an answer if possible.
